crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/integration/CrownCourtProceedingIntegrationTest.java

```diff
@@ -95,7 +100,7 @@ void givenValidContent_whenApiResponseIsError_thenProcessRepOrderIsFails() throw
                         HttpMethod.POST, objectMapper.writeValueAsString(apiProcessRepOrderRequest), ENDPOINT_URL))
                 .andExpect(status().is5xxServerError())
                 .andExpect(content().contentType(MediaType.APPLICATION_JSON))
-                .andExpect(jsonPath("$.message").value(ERROR_MSG));
+                .andExpect(jsonPath("$.message").value("500 Internal Server Error from GET http://localhost:9998/api/internal/v1/assessment/ioj-appeal/repId/" + TEST_REP_ID + "/current-passed"));
```

Comment:
Duplication here, it would be good if we didn't have to build this string twice for every test case:
``
var stubUrl = "http://localhost:" + wiremock.port() + "/api/internal/v1/assessment/ioj-appeal/repId/" + TEST_REP_ID + "/current-passed";
``
There is also a `wiremock.baseUrl()` method, but this default to the https port and caused me some issues